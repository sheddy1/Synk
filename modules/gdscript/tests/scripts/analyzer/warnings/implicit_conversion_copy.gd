func untyped_func(_x: Variant) -> void:
	pass

func array_func(_x: Array) -> void:
	pass

func typed_array_func(_x: Array[int]) -> void:
	pass

func test():
	var pba := PackedByteArray()
	untyped_func(pba)
	array_func(pba)
	typed_array_func(pba)

