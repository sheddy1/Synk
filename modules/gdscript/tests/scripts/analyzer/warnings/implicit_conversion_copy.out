GDTEST_OK
>> WARNING
>> Line: 13
>> IMPLICIT_CONVERSION_CAUSES_COPY
>> Implicit conversion from "PackedByteArray" to "Array" causes value to be copied.
>> WARNING
>> Line: 14
>> IMPLICIT_CONVERSION_CAUSES_COPY
>> Implicit conversion from "PackedByteArray" to "Array" causes value to be copied.
